src/integration/pages/mediaAssetPage/crossPlatformTests.js

```diff
@@ -23,23 +23,41 @@ export default () => {
     expect(timestampEl.textContent).toMatchSnapshot();
   });
 
-  const bulletedListEl = document.querySelector('main ul[role="list"]');
+  const bulletedListEl = document.querySelector('main ul[role="list"] > li');
 
   if (bulletedListEl) {
-    it('I can see the bulleted list item', () => {
+    it('I can see a bulleted list item', () => {
       expect(bulletedListEl).toBeInTheDocument();
       expect(bulletedListEl.textContent).toBeTruthy();
       expect(bulletedListEl.textContent).toMatchSnapshot();
     });
   }
 
-  const releatedContentEl = document.querySelector('section [role="list"]');
+  const bulletedListlink = document.querySelector(
+    'main ul[role="list"] > li > a',
+  );
 
-  if (releatedContentEl) {
-    it('I can see the related content', () => {
-      expect(releatedContentEl).toBeInTheDocument();
-      expect(releatedContentEl.textContent).toBeTruthy();
-      expect(releatedContentEl.textContent).toMatchSnapshot();
+  if (bulletedListlink) {
+    it('I can see a bulleted list item with link', () => {
+      expect(bulletedListlink.getAttribute('href')).toMatchSnapshot(
+        bulletedListlink.textContent,
+      );
+    });
+  }
+
+  const relatedContentItems = document.querySelectorAll(
+    'section [role="list"] a',
+  );
+
+  if (relatedContentItems) {
+    it('I can see related content', () => {
+      relatedContentItems.forEach(relatedContent => {
+        expect(relatedContent).toBeInTheDocument();
+        expect(relatedContent.textContent).toBeTruthy();
+        expect(relatedContent.getAttribute('href')).toMatchSnapshot(
+          relatedContent.textContent,
```

Comment:
Just a name suggestion so it's clear what is stored in the variables.

```suggestion
  const relatedContentLinks = document.querySelectorAll(
    'section [role="list"] a',
  );

  if (relatedContentLinks) {
    it('I can see related content', () => {
      relatedContentLinks.forEach(relatedContentLink => {
        expect(relatedContentLink).toBeInTheDocument();
        expect(relatedContentLink.textContent).toBeTruthy();
        expect(relatedContentLink.getAttribute('href')).toMatchSnapshot(
          relatedContent.textContent,
```